_studio/mfx_lib/encode_hw/h264/src/mfx_h264_encode_hw.cpp

```diff
@@ -1993,8 +2000,16 @@ void ImplementationAvc::OnEncodingQueried(DdiTaskIter task)
 
     if (m_useMbControlSurfs && task->m_isMBControl)
         ReleaseResource(m_mbControl, task->m_midMBControl);
-
-
+#if defined(MXF_ENABLE_MCTF_IN_AVC)
+    if (IsMctfSupported(m_video) && task->m_midMCTF)
+    {
+        ReleaseResource(m_mctf, task->m_midMCTF);
+        task->m_midMCTF = MID_INVALID
+        MfxHwH264Encode::Zero(task->m_handleMCTF);
```

Comment:
Lines 2007 and 2008 are invalid in C++ language!

Reply:
I just pushed that change